src/current/v23.2/molt-fetch.md

```diff
@@ -99,6 +103,12 @@ Complete the following items before using MOLT Fetch:
 
 - If a MySQL database is set as a [source](#source-and-target-databases), the [`--table-concurrency`](#global-flags) and [`--export-concurrency`](#global-flags) flags **cannot** be set above `1`. If these values are changed, MOLT Fetch returns an error. This guarantees consistency when moving data from MySQL, due to MySQL limitations. MySQL data is migrated to CockroachDB one table and shard at a time, using [`WITH CONSISTENT SNAPSHOT`](https://dev.mysql.com/doc/refman/8.0/en/commit.html) transactions.
 
+- To prevent memory outages during data export of tables with large rows, estimate the amount of memory used to export a table: `--row-batch-size * --export-concurrency * average size of the table rows`. If you are exporting more than one table at a time (i.e., [`--table-concurrency`](#global-flags) is set higher than `1`), add the estimated memory usage for the tables with the largest row sizes. Ensure that you have sufficient memory to run `molt fetch`, and adjust `--row-batch-size` accordingly.
```

Comment:
```suggestion
- To prevent memory outages during data export of tables with large rows, estimate the amount of memory used to export a table: 

    {% include_cached copy-clipboard.html %}
    ~~~ shell
    --row-batch-size * --export-concurrency * average size of the table rows
    ~~~
    
    If you are exporting more than one table at a time (i.e., [`--table-concurrency`](#global-flags) is set higher than `1`), add the estimated memory usage for the tables with the largest row sizes. Ensure that you have sufficient memory to run `molt fetch`, and adjust `--row-batch-size` accordingly.
```

I think the formula is easier to read when it's in a block. I didn't verify the formatting locally.